src/chapters/thymeleaf-views/template-conditionals.rst

```diff
@@ -0,0 +1,232 @@
+``false``, then Thymeleaf does NOT generate the element, and the content stays
+off the page.
+
+.. admonition:: Example
+
+   Assume that ``coffeeOptions`` and ``userSelection`` represent an ArrayList
+   and String, respectively, that are passed in by the controller.
+
+   .. sourcecode:: html
+      :linenos:
+
+      <ol th:if = "${coffeeOptions.size() > 1}">
+         <li th:each="item : ${coffeeOptions}" th:text="${item}"></li>
+      </ol>
+
+      <h2 th:if = '${userSelection.equals("instant")}'>You can do better!</h2>
+
+The conditional in line 1 checks that ``coffeeOptions`` contains more than one
+item. If ``true``, then the ordered list is rendered in the view. The
+``th:if`` statement in line 5 compares a user's flavor choice to the string
+``"instant"``. If they match, then Thymeleaf adds the heading element to the
+view.
+
+Adding vs. Displaying/Hiding
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+``th:if`` determines if content is *added or not added* to a page. This is
+different from deciding if content should be *displayed or hidden*.
+
+*Hidden* content still occupies space on a page and requires some amount of
+memory. When ``th:if`` evaluates to ``false``, content remains absent from the
+page---requiring neither space on the page nor memory. This is an important
+consideration when including items like images or videos on your website.
+
+What is ``true``?
+^^^^^^^^^^^^^^^^^^
+
+The ``th:if = "${condition}"`` attribute can evaluate more than simple boolean
+variables and statements. It will also return ``true`` according to these
+rules:
+
+#. If ``condition`` is a boolean or statement and ``true``.
+#. If ``condition`` is a non-zero number or character.
+#. If ``condition`` is a string that is NOT ``"false"``, ``"off"``, or
+   ``"no"``.
+#. If ``condition`` is a data type other than a boolean, number, character, or
+   String.
+
+``th:if`` will evaluate to ``false`` whenever ``condition`` is ``null``.
+
+``unless`` Instead of ``else``
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+In Java, we use an ``if/else`` structure to have our code execute certain steps
+when a variable or statement evaluates to ``true`` but a different set of steps
+for a ``false`` result. Thymeleaf provides a similar option with ``th:unless``:
+
+.. sourcecode:: groovy
+
+   th:unless = "${condition}"
+
+Just like ``th:if``, the ``th:unless`` attribute gets placed inside an HTML
+tag. In this case, however, Thymeleaf adds the HTML element to the webpage when
+``condition`` evaluates to ``false``.
+
+We could update our coffee code with an ``unless``:
+
+.. admonition:: Example
+
+   .. sourcecode:: html
+      :linenos:
+
+      <h2 th:unless = '${userSelection.equals("instant")}'>Excellent choice!</h2>
+
+As long as ``userSelection`` is NOT ``"instant"``, the condition evaluates to
+``false``, and the ``h2`` element gets added to the view.
+
+If we want to set up a situation like *If true, do this thing. Otherwise, do
```

Comment:
"If" does not need to be capitalized here.

Reply:
Fixed.  Thanks.